projectroles/migrations/0024_project_star_app_setting.py

```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
+
+
+def migrate_stars(apps, schema_editor):
+    """Create project_star AppSettings"""
+    ProjectUserTag = apps.get_model('projectroles', 'ProjectUserTag')
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+    for projectusertag in ProjectUserTag.objects.all():
```

Comment:
Just a minor note, it is convention to name variables with underscores between words. Also since this is just a local control structure, you can use something simple like `tag`.

Also, as I requested before: filter objects by `name=PROJECT_TAG_STARRED`, just in case.